### ISSUE TITLE
Parameter setting problem

### ISSUE THREAD

**Issue by wenfan-wenfan** [MikeG27/BCI_Image_EEG_Generator#10]
Dear Michal Gorski：
I continue to reprise your work this week, I was still stuck in the setting of training parameters. I tried to adjust the learning rate, the size of convolution kernel, the optimizer, the ratio of training set to test set, but I still couldn't get the effect picture you showed. 

What were your parameters at that time?Of course, jupyter in 3.1 _mikeG27_training.py can't work very well, in the image reconstruction above the error message.

Best wishes to you

**Comment by MikeG27** [comment]
Hi!

In general you can`t reproduce my work on validation set due to low quality
of measurements. If you want to reproduce my work please generate images
based on train set.

Best Wishes

sob., 5 gru 2020, 13:15 użytkownik wenfan-wenfan <notifications@github.com>
napisał:

**Comment by wenfan-wenfan** [comment]
Dear Michal Gorski：
Thanks for your reply ! Although it is now 2:40 am in China, I am still very happy, very excited to see your reply!

I noticed that you just wrote the prompt in "READme.md". Do you mean that I need to use eeg acquisition equipment to build a training set of my own according to the methods and steps of your experiment?

But I have downloaded the mnist-64s.csv and mnist-Flex.csv you proposed. In order to perfectly reproduce your work, can these two data sets continue to be used?

Best wishes to you
